lib/client-api/src/story_store.test.ts

```diff
@@ -793,16 +793,18 @@ describe('preview.story_store', () => {
     });
   });
 
-  describe('RENDER_CURRENT_STORY', () => {
+  describe('STORY_ERRORED', () => {
     it('is emitted when setError is called', () => {
       const onRenderCurrentStory = jest.fn();
-      channel.on(Events.RENDER_CURRENT_STORY, onRenderCurrentStory);
+      channel.on(Events.STORY_ERRORED, onRenderCurrentStory);
```

Comment:
This is semantically wrong. No story errored, there was a problem configuring the store.

We use `RENDER_CURRENT_STORY` here to tell the `StoryRenderer` to figure out what to render to the screen.